examples/rllib/multiagent_exps/multiagent_traffic_light_grid.py

```diff
@@ -0,0 +1,282 @@
+    :param n_columns:
+    :param edge_inflow:
+    :return:
+    """
+    # we place a sufficient number of vehicles to ensure they confirm with the
+    # total number specified above. We also use a "right_of_way" speed mode to
+    # support traffic light compliance
+    vehicles = VehicleParams()
+    num_vehicles = (N_LEFT + N_RIGHT) * n_columns + (N_BOTTOM + N_TOP) * n_rows
+    vehicles.add(
+        veh_id="human",
+        acceleration_controller=(SimCarFollowingController, {}),
+        car_following_params=SumoCarFollowingParams(
+            min_gap=2.5,
+            max_speed=V_ENTER,
+            decel=7.5,  # avoid collisions at emergency stops
+            speed_mode="right_of_way",
+        ),
+        routing_controller=(GridRouter, {}),
+        num_vehicles=num_vehicles)
+
+    # inflows of vehicles are place on all outer edges (listed here)
+    outer_edges = []
+    outer_edges += ["left{}_{}".format(n_rows, i) for i in range(n_columns)]
+    outer_edges += ["right0_{}".format(i) for i in range(n_rows)]
+    outer_edges += ["bot{}_0".format(i) for i in range(n_rows)]
+    outer_edges += ["top{}_{}".format(i, n_columns) for i in range(n_rows)]
+
+    # equal inflows for each edge (as dictate by the EDGE_INFLOW constant)
+    inflow = InFlows()
+    for edge in outer_edges:
+        inflow.add(
+            veh_type="human",
+            edge=edge,
+            vehs_per_hour=edge_inflow,
+            departLane="free",
+            departSpeed=V_ENTER)
+
+    flow_params = dict(
+        # name of the experiment
+        exp_tag="grid_0_{}x{}_i{}_multiagent".format(n_rows, n_columns,
+                                                     edge_inflow),
+
+        # name of the flow environment the experiment is running on
+        env_name='MultiTrafficLightGridPOEnv',
+
+        # name of the scenario class the experiment is running on
+        scenario="SimpleGridScenario",
+
+        # simulator that is used by the experiment
+        simulator='traci',
+
+        # sumo-related parameters (see flow.core.params.SumoParams)
+        sim=SumoParams(
+            restart_instance=True,
+            sim_step=1,
+            render=False,
+        ),
+
+        # environment related parameters (see flow.core.params.EnvParams)
+        env=EnvParams(
+            horizon=HORIZON,
+            additional_params={
+                "target_velocity": 50,
+                "switch_time": 3,
+                "num_observed": 2,
+                "discrete": False,
+                "tl_type": "actuated",
+                "num_local_edges": 4,
+                "num_local_lights": 4,
+            },
+        ),
+
+        # network-related parameters (see flow.core.params.NetParams and the
+        # scenario's documentation or ADDITIONAL_NET_PARAMS component)
+        net=NetParams(
+            inflows=inflow,
+            no_internal_links=False,
+            additional_params={
+                "speed_limit": V_ENTER + 5,
```

Comment:
Why is this V_ENTER + 5? If there's a reason for this magic number, we should at least add a comment here explaining it.